LiveWallpaperQuotes/app/src/main/java/com/stanleyidesis/quotograph/api/drawing/LWQDrawScript.java

```diff
@@ -185,8 +186,8 @@ void draw() {
         try {
             final int blurPreference = LWQPreferences.getBlurPreference();
             final int dimPreference = LWQPreferences.getDimPreference();
-            // Determine cache validity
-            Bitmap toDraw = blurPreference > 0f ? generateBitmap(blurPreference, backgroundImage) : backgroundImage;
+            // Blur or choose the raw background image
+            Bitmap toDraw = blurPreference > 0.5f ? generateBitmap(blurPreference, backgroundImage) : backgroundImage;
```

Comment:
Sometimes, dragging the SeekBar all-the-way to zero does not register perfectly as `0f`, his compensates for that.